app/client/models/ToggleEnterpriseModel.ts

```diff
@@ -0,0 +1,21 @@
+import {AppModel, getHomeUrl} from 'app/client/models/AppModel';
+import {Disposable, Observable} from "grainjs";
+import {ConfigAPI} from 'app/common/ConfigAPI';
+
+export class ToggleEnterpriseModel extends Disposable {
```

Comment:
What do you think about inlining those 2 methods in the `ToggleEnterpriseWidget.ts`. I think it will make it easier to understand, and will concentrate the logic for this `edition` toggle in one place. Having it in its own class raises some questions and concerns, for example:
- It accepts `_appModel` in the constructor, but it doesn't use it at all
- It doesn't explain why it needs to fetch something after a successful call that sets that value. Probably the UI needs it, but we don't see it being used here.

Reply:
I've simplified the logic according to our discussion, but I think I still like the separate model.